lib/programmemory.cpp

```diff
@@ -669,7 +669,7 @@ static ValueFlow::Value evaluate(const std::string& op, const ValueFlow::Value&
     return result;
 }
 
-using BuiltinLibraryFunction = std::function<ValueFlow::Value(const std::vector<ValueFlow::Value>&)>;
+using BuiltinLibraryFunction = std::function<ValueFlow::Value (const std::vector<ValueFlow::Value>&)>;
```

Comment:
This looks undesired.

Reply:
yes I agree it is undesired

Reply:
but imho it's not a blocker.

Reply:
I agree. I just highlighted it out so I won't forget looking into it later on.